langlib/lang.transaction/src/main/ballerina/transaction.bal

```diff
@@ -81,6 +112,19 @@ public transactional isolated function setRollbackOnly(error? e) {
 
 # Tells whether it is known that the transaction will be rolled back.
 #
+# ```
+# function createEntity() {
+#     transaction {
+#         boolean willRollBack = transaction:getRollbackOnly();
```

Comment:
Shall we print the output by adding a sample where the `getRollbackOnly` returns true?

Reply:
to show out puts `true` or `false` we have to use another langlib function: `setRollbackOnly()`. Doesn't it distract the user?

Reply:
IMO, that should be fine, since that's a dependent function.
But, make the first example a very basic one